src/applications/vaos/services/location/index.js

```diff
@@ -342,8 +344,20 @@ export async function getCommunityProvider(id) {
  * @param {Array<string>} params.siteIds A list of three digit VistA site ids
  * @returns {Array<Location>} A list of parent Locations
  */
-export async function fetchParentLocations({ siteIds }) {
+export async function fetchParentLocations({ siteIds, useV2 }) {
   try {
+    if (useV2) {
+      const parentFacilities = await getParentFacilitiesV2(siteIds);
```

Comment:
I don't think there's anything in here that limits the facilities returned to parent sites. I described that logic in the ticket description, but it looks to me like this is going to return the facilities info for the sites the user is registered at. That's different from the list of all the parent sites they have access to.

Reply:
Got it. I added filtering to the v2 transformer to avoid adding that logic to the request method.